spec/raven/event_spec.rb

```diff
@@ -347,6 +347,7 @@ def raven_context
   end
 
   describe '.initialize' do
+    # NOTE: This test is completely pointless.
```

Comment:
@nateberkopec Don't quote me on this, but I would take a look at it in the weekend, most likely.